substrate-node/pallets/pallet-tfgrid/src/lib.rs

```diff
@@ -2053,6 +2059,23 @@ pub mod pallet {
             let account_id = ensure_signed(origin)?;
             Self::_change_power_target(account_id, node_id, power_target)
         }
+
+        #[pallet::call_index(37)]
+        #[pallet::weight(100_000_000 + T::DbWeight::get().writes(1).ref_time() + T::DbWeight::get().reads(1).ref_time())]
+        pub fn bond_twin_account(origin: OriginFor<T>, twin_id: u32) -> DispatchResultWithPostInfo {
```

Comment:
Would bind_twin_account not be a better name?

Reply:
In the frame pallet staking, bond is also used. So let's try to mimic that as much as possible

Reply:
https://github.com/paritytech/substrate/blob/f5c2cecd7efe6dcbc828906794f3c4c41f23c791/frame/staking/src/lib.rs#L62